flake.nix

```diff
@@ -0,0 +1,28 @@
+{
+  description = "Tabry in Rust";
+  inputs = {
+    flake-utils.url = "github:numtide/flake-utils";
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+  };
+  outputs = { nixpkgs, flake-utils, ... }:
+    flake-utils.lib.eachDefaultSystem (
+      system:
+        let
+          pkgs = nixpkgs.legacyPackages."${system}";
+          tabry = pkgs.callPackage ./default.nix {};
+        in {
+          packages = {
+            default = tabry;
+          };
+          devShell = pkgs.mkShell {
+            packages = with pkgs; [
+              rustfmt
+              cargo
+              rustc
+            ];
+            shellHook = ''
```

Comment:
TODO here?

Reply:
Yeah, I can remove these... I just copied this from another project, and I leave this blank because I always end up forgetting the name of the attribute 😅